src/Compilers/CSharp/Test/Symbol/DocumentationComments/CrefTests.cs

```diff
@@ -6660,5 +6660,49 @@ void S()
             var parameterSymbol = ((MethodSymbol)model.GetSymbolInfo(cref).Symbol).Parameters.Single();
             Assert.Equal(RefKind.In, parameterSymbol.RefKind);
         }
+
+        [Fact]
+        public void Cref_TupleType()
+        {
+            var source = @"
+using System;
+/// <summary>
+/// See <see cref=""ValueTuple{T,T}""/>.
+/// </summary>
+class C
+{
+}
+";
+            var compilation = CreateCompilationWithDocumentationComments(source);
+            var cMember = compilation.GetMember<NamedTypeSymbol>("C");
+            var xmlDocumentationString = cMember.GetDocumentationCommentXml();
+
+            var xml = System.Xml.Linq.XDocument.Parse(xmlDocumentationString);
+            var cref = xml.Descendants("see").Single().Attribute("cref").Value;
+
+            Assert.Equal("T:System.ValueTuple`2", cref);
+        }
+
+        [Fact]
+        public void Cref_TupleTypeField()
```

Comment:
Does testing the methods off of ValueTuple expose any interesting behavior, or is it exactly like fields?

Reply:
It exhibits the same behavior; the crash happens when you dot into the tuple because it ends up binding a QualifiedCref rather than a MemberCref. That in turn requires resolving the generic type which causes the TransformToTupleIfCompatible in Binder_Symbols.cs.